### DISCUSSION TITLE
등록 메일 재전송 여부

### DISCUSSION THREAD

**Discussion by jianny-lee** [devrel-kr/HackaLearn#47] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요! 제가 참가를 등록했는데 메일이 오지 않았습니다.
확인 부탁드립니다
팀명 : Konggg

감사합니다.

**Comment by kooljay82** [comment]
스팸도 확인 해보셨나요?? @justinyoo 확인 부탁드립니당!!!

**Reply by jianny-lee** [reply]
네 스팸도 확인 해봤는데 안 왔습니다.

**Comment by justinyoo** [comment, marked as answer]
백엔드 쪽에서 홀드가 됐었네요. 지금 갔을 거예요. 다시 한 번 확인해 보세욥.

**Reply by jianny-lee** [reply]
> 백엔드 쪽에서 홀드가 됐었네요. 지금 갔을 거예요. 다시 한 번 확인해 보세욥.

아 네 메일 받았습니다! 감사합니다!!
